catalyst-gateway/event-db/Earthfile

```diff
@@ -4,6 +4,26 @@
 
 VERSION 0.7
 
+# cspell: words pgdg eventdb dbname 
+
+ADD_PSQL:
+    COMMAND
+
+    RUN apt-get update && apt-get install -y --no-install-recommends \
+        wget \
+         gnupg2 \
+         ca-certificates \
+         lsb-release \
+         apt-transport-https \
+         && rm -rf /var/lib/apt/lists/* && \
+         wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+         echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+         apt-get update && \
+         apt-get install -y --no-install-recommends postgresql-14 && \
+         apt-get remove -y wget gnupg2 lsb-release apt-transport-https && \
+         apt-get autoremove -y && \
+         apt-get clean && rm -rf /var/lib/apt/lists/*
+
 build:
     FROM python:3.11-slim-bullseye # Use a debian stable with python
```

Comment:
This should use the `postgres-base` container I discussed above.
Looks like pthon is already part of the upstream postgres container so no need to add it.  Only add dependencies to the base container when something breaks because the command it need is missing.